### PR TITLE
feat: update payload for installment credit card

### PR DESCRIPTION
**Pull request by Grace-07**
https://developers.paymongo.com/docs/card-installments-api

### CODE REVIEW

File: src/PaymentIntent.php [hunk 1]
```diff
@@ -14,7 +14,7 @@ class PaymentIntent extends PaymongoClient {
      * @param  mixed $metadata
      * @return mixed
      */
-    public function create($amount, $payment_method_allowed, $description = null, $metadata = null, $currency = 'PHP') {
+    public function create($amount, $payment_method_allowed, $payment_method_options, $description = null, $metadata = null, $currency = 'PHP') {
```
Comment (micahbule) [inline src/PaymentIntent.php]: if `payment_method_options` can be null as per code below, set the default value to `null`. Do the same for the `attachPaymentMethod()` method.

**Comment by micahbule** [comment]
Additionally, please run `composer run-script test` to run the unit tests. Fix any errors that occur.

File: src/PaymentIntent.php (hunk 1 above)
Comment (Grace-07) [reply src/PaymentIntent.php]: Done

**Review by micahbule — APPROVED** [review]

**Comment by micahbule** [comment]
Please create a PR from the `main` branch of your fork. Thanks.

**Pull request closed without merging**